editor/src/components/editor/global-shortcuts.tsx

```diff
@@ -733,42 +737,72 @@ export function handleKeyDown(
         return actions
       },
       [TOGGLE_TEXT_BOLD]: () => {
-        return isSelectMode(editor.mode)
-          ? editor.selectedViews.map((target) => {
-              const element = MetadataUtils.findElementByElementPath(editor.jsxMetadata, target)
-              return toggleTextBold(target, element?.specialSizeMeasurements.fontWeight ?? null)
-            })
-          : []
+        if (isSelectMode(editor.mode)) {
+          let first = true
+          editor.selectedViews.forEach((target) => {
```

Comment:
You could get rid of the `let first` here by using `.forEach((target, index) => {`

Reply:
good idea, done